lib/flipper/ui.rb

```diff
@@ -45,7 +45,7 @@ def self.app(flipper, options = {})
       builder.use Middleware, flipper
       builder.run app
       klass = self
-      builder.define_singleton_method(:inspect) { klass } # pretty rake routes output
+      builder.define_singleton_method(:inspect) { klass.to_s } # pretty rake routes output
```

Comment:
totally, or maybe even just `self.inspect`? and then don't need the reference w/ klass